beamer/data/agent.conf.example

```diff
@@ -25,6 +25,13 @@ rpc-url = "GOERLI_OPTIMISM_RPC_URL"
 # Each token is represented by a pair [chain-id, token-address].
 # All tokens within the same list are considered equivalent and
 # transfers between them are allowed.
+# A third value per token representation is optional, which defines
+# the allowance amount, the agent will approve to the fill manager contract.
+# [chain-id, token-address, allowance]
+# Allowed values are:
+# - Any value > 0, which defines the exact allowance given
+# - -1, the agent will approve type(uint256).max
```

Comment:
How about using TOML-supported infinity value, `inf` e.g.
```toml
        ["11", "0x2644292EE5aed5c17BDcc6EDF1696ba802351cf6", inf]
```
which would in Python be converted to `math.inf`?

Reply:
I gave my explanation above why I chose -1, and actually inf would also semantically not be correct as more recent versions of ERC20 don't support unlimited allowance, only set it to the max value, but handle it as any other value (subtracting the amounts on each transfer)